registry/storage/driver/walk.go

```diff
@@ -32,7 +34,14 @@ func WalkFallback(ctx context.Context, driver StorageDriver, from string, f Walk
 		// performance bottleneck.
 		fileInfo, err := driver.Stat(ctx, child)
 		if err != nil {
-			return err
+			switch err.(type) {
+			case storageDriver.PathNotFoundError:
```

Comment:
`s/storageDriver\.//` and then we should be good to go now that this file is in the driver package. Also I'll see if I can add a test for this before we merge.